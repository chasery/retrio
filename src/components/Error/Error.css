.Error {
  color: #d52941;
}
