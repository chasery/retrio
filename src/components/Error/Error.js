import React from 'react';
import './Error.css';

function Error(props) {
  const { message } = props;
  return <p className='Error'>{message}</p>;
}

export default Error;
