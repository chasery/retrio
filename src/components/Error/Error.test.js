import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Error from './Error';

describe('Error component', () => {
  it('renders without error', () => {
    const wrapper = shallow(<Error />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
