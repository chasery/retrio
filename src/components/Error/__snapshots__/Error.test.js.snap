// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component renders without error 1`] = `
<p
  className="Error"
/>
`;
